samples/azure-quantum/dynamic-error-correction/DynamicErrorCorrection.ipynb

```diff
@@ -0,0 +1,369 @@
+        "    elif (parity01 == Zero and parity12 == One) {\n",
+        "        X(register[2]);\n",
+        "    }\n",
+        "\n",
+        "    return parity01 == One or parity12 == One;\n",
+        "}"
+      ]
+    },
+    {
+      "attachments": {},
+      "cell_type": "markdown",
+      "metadata": {
+        "nteract": {
+          "transient": {
+            "deleting": false
+          }
+        }
+      },
+      "source": [
+        "## 6. Define the DynamicBitFlipCode operation\n",
+        "Take all of the operations defined above and iterate multiple times:\n",
+        "1. Apply a logical operation that may introduce errors.\n",
+        "2. Attempt to correct introduced error and increment error count accordingly.\n",
+        "3. Measure the register to read the logical qubit value.\n",
+        "4. Report the value of each logical qubit."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {},
+      "outputs": [],
+      "source": [
+        "operation DynamicBitFlipCode() : (Result, Int){\n",
+        "    // Create a registers that represents a logical qubit.\n",
+        "    use localRegister = Qubit[3];\n",
+        "\n",
+        "    // Apply several unitary operations to the encoded qubits performing error correction between each application.\n",
+        "    mutable corrections = 0;\n",
+        "    within {\n",
+        "        // Encode/Decode logical qubit.\n",
+        "        Encode(localRegister);\n",
+        "    }\n",
+        "    apply {\n",
+        "        LogicalX(localRegister);\n",
+        "        let iterations = 5;\n",
+        "        for _ in 1 .. iterations {\n",
+        "            // Apply unitary operations.\n",
+        "            ApplyLogicalOperation(localRegister);\n",
+        "\n",
+        "            // Perform error correction and increase the counter if a correction was made.\n",
+        "            let (parityA01, parityA12) = MeasureSyndrome(localRegister);\n",
+        "            let correctedError = CorrectError(localRegister, parityA01, parityA12);\n",
+        "            if (correctedError) {\n",
+        "                set corrections += 1;\n",
+        "            }\n",
+        "        }\n",
+        "    }\n",
+        "\n",
+        "    // Measure the first qubit in register and return value.\n",
+        "    let result = MResetZ(localRegister[0]);\n",
+        "    ResetAll(localRegister);\n",
+        "    return (result, corrections);\n",
+        "}"
+      ]
+    },
+    {
+      "attachments": {},
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+        "## 7. Submit the DynamicBitFlipCode operation\n",
+        "For 50 shots, as configured below, the following will use about 11.31 units (HQCs or eHQCs).<br/>\n",
+        "If the job is not completed after the timeout (300 seconds), you can query the status followed by querying the output.<br/>\n",
```

Comment:
```suggestion
        "If the job is not completed after the client timeout (300 seconds), you can query the status using %azure.status followed by querying the %azure.output.<br/>\n",
```

Reply:
Also %azure.status and %azure.output will only work if they are using the same kernel instance. If kernel was restarted, they will need to use %azure.status <job_id> to get their job.

Reply:
@xinyi-joffre added the need to provide a job ID if the kernel was restarted to the description.